gxa/src/main/webapp/WEB-INF/jsp/tiles/views/home/release-stats.jsp

```diff
@@ -0,0 +1,26 @@
+<%--@elvariable id="info" type="String"--%>
+<%--@elvariable id="numberOfSpecies" type="Number"--%>
+<%--@elvariable id="numberOfStudies" type="Number"--%>
+<%--@elvariable id="numberOfAssays" type="Number"--%>
+<%--@elvariable id="genomes" type="String"--%>
+<%--@elvariable id="paraSite" type="String"--%>
+<%--@elvariable id="ensembl" type="String"--%>
+<%--@elvariable id="efo" type="String"--%>
+
+<%@ page contentType="text/html;charset=UTF-8" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+
+<div class= "small-12 medium-6 columns text-left">
```

Comment:
I think we could get away with making this text a little smaller. If you view the page on a normal-sized screen (laptop), the text takes up a second line.

<img width="1677" alt="Screenshot 2019-03-22 at 14 32 57" src="https://user-images.githubusercontent.com/5923538/54829936-6a6c9700-4caf-11e9-8707-c859675844bc.png">

Reply:
Yes, I noticed it too when I saw it on my laptop. I think `<h4><small>` will work. Thanks for reminding me.